src/scanner/present.rs

```diff
@@ -240,14 +258,39 @@ async fn answer_active_oracle(
             }
         };
 
-        let receipt: Option<ethers::types::TransactionReceipt> = match tx.await {
+        let mut db_connection = match db_connection_pool
+            .get()
+            .context("could not get new connection from pool")
+        {
+            Ok(db_connection) => db_connection,
+            Err(error) => {
+                tracing::error!(
+                    "could not get database connection while trying to delete oracle from database - {}",
```

Comment:
The logged error here should refer to the `answer_tx_hash` update operation, and not the oracle delete, no?

Reply:
It should have been addressed in dab047b95b3723ab37c78f7f4ee2f97179aff9da